apps/web/src/pages/settings/organization/OrganizationLogo.tsx

```diff
@@ -0,0 +1,80 @@
+import { errorMessage, successMessage } from '@novu/design-system';
+import { IResponseError } from '@novu/shared';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useUpdateOrganizationBranding } from '../../../api/hooks';
+import { getSignedUrl } from '../../../api/storage';
+import { ProfileImage } from '../../../components/shared';
+
+type FormValues = {
+  logo: string;
+};
+
+const MIME_TYPES = {
+  'image/jpeg': 'jpeg',
+  'image/png': 'png',
+};
+
+export function OrganizationLogo({ logo }: { logo?: string }) {
+  const { updateOrganizationBranding } = useUpdateOrganizationBranding();
+  const { control, reset } = useForm<FormValues>({
+    defaultValues: {
+      logo: logo,
+    },
+  });
+
+  const { mutateAsync: getSignedUrlAction } = useMutation<
+    { signedUrl: string; path: string; additionalHeaders: object },
+    IResponseError,
+    string
+  >(getSignedUrl);
+
+  async function handleUpload(event: React.ChangeEvent<HTMLInputElement>) {
+    const file = event.target.files ? event.target.files[0] : null;
```

Comment:
```suggestion
    const file = event.target.files?.[0] ? event.target.files[0] : null;
```
In the event we can encounter an empty array